### DISCUSSION TITLE
Multi-user annotation capability in Annotation Queues

### DISCUSSION THREAD

**Discussion by juanmleng** [langfuse/langfuse#4348] [Ideas]
### Describe the feature or potential improvement

Hi team, I’m exploring the annotation functionality, specifically regarding the Score Configs support for annotation. While I understand that I can add annotations to each LLM trace, I would like to know if it’s possible to enable multiple users to annotate the same trace independently, using the same set of labels. This feature would allow us to collect multiple perspectives from different experts, to check how they annotate the same labels.

**Comment by marcklingen** [comment]
Thanks for sharing! I will ping Kelly from the townhall to share her workaround and thoughts here as well

**Reply by ahmetkaanozbek** [reply]
It’s still not possible for multiple users to annotate the same trace. It would be great to have this feature so we can compare different people’s perspectives on the same output.

**Comment by kking67** [comment]
Hey, sounds like it may not directly meet your need Juan (labeling of score configs by separate annotators), but if useful, we have explored score configs with naming conventions to align with the responsible/assigned annotator - ie., same scoring rule/set up, with initials denoting who is assigned. Then, filtering the traces such that you can see comparison across users.  I definitely still see validity in having separate annotators/tags, but this has met our needs thus far. We are also running LLM evals that follow the same series of guidelines as human annotation, and building cues based on LLM fails for human annotation - which has worked really well thus far, thanks Marc and team for pushing forward! Excited for what's to come. (note, I am not a developer, I am a domain expert collaborating with our more tech-inclined teams. Appreciate how accessible this is!)

**Reply by marcklingen** [reply]
Awesome, thanks for sharing how you implemented it through configurations for each reviewer. While this approach doesn't solve the problem at scale, I can see how it is very practical as long as the team of reviewers is small enough to manage these score configurations efficiently.

Good to have this here as a potential workaround for anyone looking for this until the feature is implemented.

**Reply by juanmleng** [reply]
Thank you Kelly! I like the idea of naming score configs to match annotators. Also, thanks for sharing your approach to refining evals criteria pairing LLMs and humans.

**Comment by JoeGosse** [comment]
Adding my vote here - the lack of multi-user scoring really hurts the value proposition of using Langfuse for human annotation over other approaches (such as spreadsheets)

**Reply by philcicle** [reply]
+1 to this. It's one of the very few things keeping my team from fully switching to Langfuse.

**Comment by lweisbord-apolitical** [comment]
This is so necessary for any meaningful evaluation projects!!!! Using workarounds now but it doesn't scale :(
